harnessed_jobs/rtm_aliveness_exposure/v0/rtm_aliveness_exposure.schema

```diff
@@ -0,0 +1,9 @@
+# -*- python -*-
+{
+    'schema_name' : 'rtm_aliveness_exposure',
+    'schema_version' : 0,
+    'image_type' : str,
+    'exptime' : int,
+    'slot' : str,
+    'bad_channels' : int
+}
```

Comment:
I don't know much about the job harness. Would be interested in a quick overview on what this schema file does, how it is related/used by the producer/validator. I will be especially interested in learning how the actual ccs job passes data/information to the validator script.

But I realize this is beyond the scope of this review, so maybe it could be done privately as not to waste other people's time.